pkg/manager/internal.go

```diff
@@ -132,6 +136,26 @@ func (cm *controllerManager) GetRecorder(name string) record.EventRecorder {
 	return cm.recorderProvider.GetEventRecorderFor(name)
 }
 
+func (cm *controllerManager) SetControllerReference(owner, object v1.Object) error {
```

Comment:
It feels a bit awkward to add a helper method on `controllerManager` for this. I get the reasoning that this needs access to scheme which is only available at mgr level. Another way would be have this helper as a function which takes scheme as input (not elegant, but isn't too bad either), WDYT ?  (thinking along the lines of `io` and `ioutil`)